doc/examples_sphinx-gallery/stochastic_variability.py

```diff
@@ -0,0 +1,93 @@
+"""
+.. _tutorials-stochastic-variability:
+
+=========================================================
+Stochastic Variability in Community Detection Algorithms
+=========================================================
+
+This example demonstrates the variability of stochastic community detection methods by analyzing the consistency of multiple partitions using similarity measures (NMI, VI, RI) on both random and structured graphs.
```

Comment:
Please spell out the names of similarity measures. If you like, you can add the abbreviations in parentheses.